src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -203,7 +203,11 @@ export class CernerCallToAction extends Component {
                   Choose the right health portal
                 </h3>
                 <p className="vads-u-font-weight--bold">
-                  {`To ${fillins.cta1} these facilities, go to My VA Health:`}
+                  {`To ${fillins.cta1} ${
```

Comment:
Should this still have the `:` at the end?

Reply:
good catch!